set-GET-beta/.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+.project
+*.swp
+*.env
```

Comment:
This is duplicate code so not needed I don't think. The same duplicate file is at the top level.  You only need 1 gitignore per repository.